client/src/utils.tsx

```diff
@@ -0,0 +1,16 @@
+export const formatBytes = (bytes: number, decimals = 2) => {
```

Comment:
a file called utils is too general. As a slight improvement create a file called fileHelper in the directory where there are other similar helpers.